### PR TITLE
[MINOR] fix: Remove newlines in spark tutorial output

### PR DESCRIPTION
**Pull request by 1ambda**
### What is this PR for?

fix: Remove newlines in spark tutorial output. I attached screenshots.

### What type of PR is it?
[Bug Fix]

### Todos

DONE

### What is the Jira issue?

MINOR

### How should this be tested?

1. Check out this PR. 
2. Build
3. Open the `Zeppelin Tutorial / Basic Features(Spark)` notebook
4. Check the output.

#### Before

image

#### After

image

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO

### CODE REVIEW

File: notebook/2A94M5J1Z/note.json [hunk 1]
```diff
@@ -367,7 +440,25 @@
   "name": "Zeppelin Tutorial/Basic Features (Spark)",
   "id": "2A94M5J1Z",
   "angularObjects": {
-    "2C73DY9P9:shared_process": []
+    "2CHK9ZDZT:shared_process": [],
```
Comment (Leemoonsoo) [inline notebook/2A94M5J1Z/note.json]: Shell we not have these changes here?

File: notebook/2A94M5J1Z/note.json [hunk 2]
```diff
@@ -197,34 +245,57 @@
...
           }
         ],
         "enabled": true,
         "editorSetting": {
-          "language": "sql",
+          "language": "text",
           "editOnDblClick": false
         },
-        "editorMode": "ace/mode/sql"
+        "editorMode": "ace/mode/text",
```
Comment (felixcheung) [inline notebook/2A94M5J1Z/note.json]: why the changes to editorMode - scala -> text and here sql -> text?

**Comment by 1ambda** [comment]
@Leemoonsoo @felixcheung Thanks for the review.

- I removed all the things which are not related to the output.
- and the paragraph in the spark tutorial now looks

image

**Comment by 1ambda** [comment]
Hi all. ping.
